netbox/dcim/api/serializers.py

```diff
@@ -619,6 +619,24 @@ class Meta:
         ]
 
 
+class CustomDeviceListSerializer(serializers.ListSerializer):
+    def validate(self, attrs):
+        validated_data = super().validate(attrs)
+        devices = [Device(**item) for item in validated_data]
+        # break these up into racks for space validation
+        racks = {}
+        for device in devices:
+            if device.rack and device.position is not None:
+                if device.rack not in racks:
+                    racks[device.rack] = []
+                racks[device.rack].append(device)
+
+        for rack, devices in racks.items():
+            units = rack.check_for_space(devices)
```

Comment:
```suggestion
            rack.check_for_space(devices)
```

`check_for_space` does not have a return value and `units` is an unused variable.